numpy/lib/utils.py

```diff
@@ -1136,7 +1139,6 @@ def safe_eval(source):
     """
     # Local import to speed up numpy's import time.
     import ast
-
```

Comment:
I think you can put this deprecation back, but have to replace all use-cases with the `ast.literal_eval(source)` call that this does. Maybe the DeprecationWarning should mention to use `ast.literal_eval` instead.

Reply:
@seberg , can I try deprecating `self_eval` in a different pull request altogether, if that is alright?

Reply:
Yes of course, but should probably remove the documentation change as well for now then.

Reply:
done